evaluation_plans/osps/quality/steps.go

```diff
@@ -246,26 +242,31 @@ func isManifestFile(filename string, pattern string) bool {
 	return filename == pattern
 }
 
-func verifyDependencyManifests(payloadData interface{}) (layer4.Result, string) {
+func countDependencyManifests(payloadData interface{}) (result layer4.Result, message string) {
 	data, message := reusable_steps.VerifyPayload(payloadData)
 	if message != "" {
 		return layer4.Unknown, message
 	}
 
+	manifestsCount := data.DependencyManifestsCount
+	if manifestsCount > 0 {
+		return layer4.Passed, fmt.Sprintf("Found %d dependency manifests from GitHub API", manifestsCount)
+	}
+
+	// Note: I'm not sure whether the rest of this logic will ever be needed as GitHub is detecting the manifests automatically
+	// Also, the repository tree is limited to three levels deep currently, so this isn't guaranteed to find all manifests
```

Comment:
If in the future we provide the ability to run checks against a git repository on the local filesystem (common use case for github actions workflows and for maintainer/contributor validation of changes as they work to comply) I can imagine it would be helpful to support our own repository search for manifests.

But, if GitHub's API can reliably report the manifests as-is, I'd argue we should actively avoid alternative implementations (like this one below) until such time as we support filesystem evaluation